ui/src/App.jsx

```diff
@@ -65,13 +65,27 @@ const App = ({ signOut }) => {
     }
   }
 
+  const submitPromptUpdate = async () => {
+    const newPromptList = promptList.map((item) => {
+      if (item.label === promptView.label) {
+        return { label: promptView.label, value: promptView.value }
+      } else {
+        return item
+      }
+    })
+    const response = await API.post("api", "/update-prompt", { body: { promptView }, headers: { "Content-Type": "text/plain" } });
+    console.log("`Updated Prompt List: ", newPromptList);
```

Comment:
You could use `console.info` here to reduce the noise in the user's console unless they explicitly wish to see the logs